lib/src/browser/disposable_browser.dart

```diff
@@ -159,9 +159,9 @@ class Disposable implements disposable_common.Disposable {
   /// generate a [LeakFlag] object. This can be used to generate a [LeakFlag]
   /// and manually attach it to an object. For example, this may be useful in
   /// code transpiled to JavaScript from another language.
-  static void enableDebugMode({bool logging: true, bool telemetry: true}) {
+  static void enableDebugMode({bool disableLogging, bool disableTelemetry}) {
```

Comment:
It'd be good to still default these, and I prefer positive boolean names. Negating negative names gets weird. `if (enableLogging)` is easier to read than `if (!disableLogging)`.

Reply:
The default is "logging is enabled", so the positive action being taken is to specifically disable it. At least that's how I think about it in a case like this. Also, default values in Dart are kind of lame because someone can still pass `null` directly (intentionally or otherwise).

Reply:
This isn't an action, though, it's a bool. If it were a function `disableLogging()`, that'd be fine, but for bools I'm firmly of the belief that, without a very good reason, they should be positive in nature.

Reply:
I'm of the firm belief that this is a very good reason :-)